docs/en/integrations/data-ingestion/data-formats/parquet-arrow-avro-orc.md

```diff
@@ -0,0 +1,309 @@
+
+## Working with Parquet data
+
+Parquet is an efficient file format to store data in a column-oriented way.
+
+### Importing from Parquet
+
+Before loading data, we can use [file()](/docs/en/sql-reference/functions/files.md/#file) function to explore an [example parquet file](assets/data.parquet) structure:
+
+```sql
+DESCRIBE TABLE file('data.parquet', Parquet)
+```
+
+We've used [Parquet](/docs/en/interfaces/formats.md/#data-format-parquet) as a second argument, so ClickHouse knows the file format. This will print columns with the types:
+
+```bash
+┌─name─┬─type─────────────┬─default_type─┬─default_expression─┬─comment─┬─codec_expression─┬─ttl_expression─┐
+│ path │ Nullable(String) │              │                    │         │                  │                │
+│ date │ Nullable(String) │              │                    │         │                  │                │
+│ hits │ Nullable(Int64)  │              │                    │         │                  │                │
+└──────┴──────────────────┴──────────────┴────────────────────┴─────────┴──────────────────┴────────────────┘
+```
+
+We can also explore files before actually importing data using all power of SQL:
+
+```sql
+SELECT *
+FROM file('data.parquet', Parquet)
+LIMIT 3;
+
+┌─path──────────────────────┬─date───────┬─hits─┐
+│ Akiba_Hebrew_Academy      │ 2017-08-01 │  241 │
+│ Aegithina_tiphia          │ 2018-02-01 │   34 │
+│ 1971-72_Utah_Stars_season │ 2016-10-01 │    1 │
+└───────────────────────────┴────────────┴──────┘
+```
+
+### Importing to an existing table
+
+Let's create a table to import parquet data to:
+
+```sql
+CREATE TABLE sometable
+(
+    `path` String,
+    `date` Date,
+    `hits` UInt32
+)
+ENGINE = MergeTree
+ORDER BY (date, path)
+```
+
+Now we can import data using a `FROM INFILE` clause:
+
+
+```sql
+INSERT INTO sometable
+FROM INFILE 'data.parquet' FORMAT Parquet;
+
+SELECT *
+FROM sometable
+LIMIT 5;
+
+┌─path──────────────────────────┬───────date─┬─hits─┐
+│ 1988_in_philosophy            │ 2015-05-01 │   70 │
+│ 2004_Green_Bay_Packers_season │ 2015-05-01 │  970 │
+│ 24_hours_of_lemans            │ 2015-05-01 │   37 │
+│ 25604_Karlin                  │ 2015-05-01 │   20 │
+│ ASCII_ART                     │ 2015-05-01 │    9 │
+└───────────────────────────────┴────────────┴──────┘
+```
+
+Note how ClickHouse automatically converted parquet strings (in the `date` column) to the `Date` type. This is because ClickHouse does typecast automatically based on the types in the target table.
+
+
+### Creating new tables from parquet files
+
+Since ClickHouse reads parquet file schema, we can create tables on the fly:
+
+```sql
+CREATE TABLE imported_from_parquet
```

Comment:
@DanRoscigno mention no cloud support ?

Reply:
will it work if switched to a URL for the parquet file?